.circleci/config.yml

```diff
@@ -19,21 +29,49 @@ jobs:
 
     steps:
       - checkout
-      - run: 
-          name: update-npm
-          command: 'sudo npm install -g npm@latest'
 
+      - run:
+          name: install mysql
```

Comment:
can this be in your dependencies, installed with npm install?

Reply:
That's separate from npm; this is setting up my database for real-data testing on CircleCI